scripts/travis.sh

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env bash
```

Comment:
Another option is remove "accept license" function from apollo_base.sh. I don't think it is good to put it there.